Source/MoreDotNet.Test/Extensions/Common/XmlExtensions/XmlExtensionsTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.XmlExtensions
+{
+    using System;
+
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class XmlExtensionsTests
+    {
+        [Fact]
+        public void XmlSerialize_GivenNullArgument_ShouldThrowException()
+        {
+            object testObject = null;
+
+            Assert.Throws<ArgumentNullException>(() => testObject.XmlSerialize());
+        }
+
+        [Fact]
+        public void XmlSerialize_GivenNonNullArgument_ShouldReturnSerialized()
+        {
+            object testObject = new object();
+
+            Assert.NotNull(testObject.XmlSerialize());
```

Comment:
This just checks if the returned string is not null, but is it truly correctly serialized? An additional check will make the test better.